src/blocks/scratch3_motion.js

```diff
@@ -56,8 +56,10 @@ Scratch3MotionBlocks.prototype.goTo = function (args, util) {
         targetX = util.ioQuery('mouse', 'getX');
         targetY = util.ioQuery('mouse', 'getY');
     } else if (args.TO === '_random_') {
-        targetX = this.runtime.constructor.STAGE_WIDTH * (Math.random() - 0.5)
-        targetY = this.runtime.constructor.STAGE_HEIGHT * (Math.random() - 0.5)
+        var stageWidth = this.runtime.constructor.STAGE_WIDTH;
+        var stageHeight = this.runtime.constructor.STAGE_HEIGHT;
```

Comment:
`this.runtime.constructor` feels a bit hacky. But requiring `../engine/runtime.js` just returns an empty object ([because it's a circular require](http://stackoverflow.com/a/23875299/4633828)), so this is all I could come up with.

I'm not sure if `.constructor` is part of ES5 though.